doc/Media2.xml

```diff
@@ -2167,6 +2167,244 @@
         </variablelist>
       </section>
     </section>
+	<section xml:id="section_k4m_bcd_yyz">
+      <title>Audio Clip Management</title>
```

Comment:
Remove Configuration suffix from commands: GetAudioClips, AddAudioClip, ... is clearer.

Reply:
I am okay with this change . We will review with other onvif members too in next week VEWG call, if everyone agrees with this change, I will do the need changes.

Reply:
Configuration suffix is removed from Audio Clip Management APIs now.